packages/terra-clinical-item-collection/tests/jest/__snapshots__/ItemCollection.test.jsx.snap

```diff
@@ -445,12 +445,14 @@ exports[`should render an ItemCollection with an end accessory 1`] = `
 
 exports[`should render an ItemCollection with custom props 1`] = `
 <div
+  aria-label="Item Collection"
```

Comment:
is this correct?

Reply:
The label is no longer applied to the div. https://github.com/cerner/terra-clinical/pull/644/commits/75438d43ea6e9d600bde3846d966c3082372e338